#55 Type hinting (#83)
IviFoundation/IviDrivers: IviDriverPython/1.0/Spec/IviDriverPython.md

Name composition:
- `<vendorPrefix>-<driverIdentifier>` Example: `myvendor-specan`

#### IVI-Python Package Type-Hinting

> The driver package shall provide type-hinting for all of its interfaces. In addition, the `py.typed` file with empty content shall be present for it top-level package.

Reviewer: ```suggestion
The driver package shall provide type-hinting for all of its interfaces. In addition, an empty file named `py.typed` shall be included at the top level of the package.
```

- Readme File (`README.md`)

> - Type hinting file (`py.typed`) in the top-level package

Reviewer: ```suggestion
- Type hinting file (`py.typed`) at the top level of the package
```
